src/components/cc-oauth-consumer/cc-oauth-consumer.js

```diff
@@ -0,0 +1,278 @@
+import { css, html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { iconRemixCheckLine } from '../../assets/cc-remix.icons.js';
+import { fakeString } from '../../lib/fake-strings.js';
+import { i18n } from '../../lib/i18n/i18n.js';
+import { skeletonStyles } from '../../styles/skeleton.js';
+import { ccLink } from '../../templates/cc-link/cc-link.js';
+import '../cc-badge/cc-badge.js';
+import '../cc-block-section/cc-block-section.js';
+import '../cc-block/cc-block.js';
+import '../cc-icon/cc-icon.js';
+import '../cc-img/cc-img.js';
+import '../cc-input-text/cc-input-text.js';
+import '../cc-notice/cc-notice.js';
+
+/**
+ * @type {OauthConsumerRight}
+ */
+const SKELETON_RIGHT = {
+  name: '??????',
+  isEnabled: true,
+  section: null,
+};
+
+/**
+ * @type {Partial<OauthConsumerStateLoaded>}
+ */
+const SKELETON_OAUTH_CONSUMER_INFO = {
+  name: fakeString(15),
+  homePageUrl: fakeString(20),
+  appBaseUrl: fakeString(20),
+  description: fakeString(45),
+  image: null,
+  rights: [
+    SKELETON_RIGHT,
+    SKELETON_RIGHT,
+    SKELETON_RIGHT,
+    SKELETON_RIGHT,
+    SKELETON_RIGHT,
+    SKELETON_RIGHT,
+    SKELETON_RIGHT,
+  ],
+  key: fakeString(20),
+  secret: fakeString(20),
+};
+
+/**
+ * @typedef {import('./cc-oauth-consumer.types.js').OauthConsumerState} OauthConsumerState
+ * @typedef {import('./cc-oauth-consumer.types.js').OauthConsumerStateLoaded} OauthConsumerStateLoaded
+ * @typedef {import('./cc-oauth-consumer.types.js').OauthConsumerRight} OauthConsumerRight
+ */
+
+/**
+ * A component displaying Access details and authorizations of a oAuth Consumer.
+ *
+ * @cssdisplay block
+ */
+export class CcOauthConsumer extends LitElement {
+  static get properties() {
+    return {
+      state: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+
+    /** @type {OauthConsumerState} Sets the state of the component. */
+    this.state = { type: 'loading' };
+  }
+
+  /**
+   * @param {string|null} name
+   * @returns {string|Node}
+   */
+  _getName(name) {
+    switch (name) {
+      case 'accessOrganisations':
+        return i18n('cc-oauth-consumer.auth.access.option.access-organisations');
+      case 'accessOrganisationsBills':
+        return i18n('cc-oauth-consumer.auth.access.option.access-organisations-bills');
+      case 'accessOrganisationsConsumptionStatistics':
+        return i18n('cc-oauth-consumer.auth.access.option.access-organisations-consumption-statistics');
+      case 'accessOrganisationsCreditCount':
+        return i18n('cc-oauth-consumer.auth.access.option.access-organisations-credit-count');
+      case 'accessPersonalInformation':
+        return i18n('cc-oauth-consumer.auth.access.option.access-personal-information');
+      case 'changePassword':
+        return i18n('cc-oauth-consumer.auth.manage.option.change-password');
```

Comment:
issue: I think we decided this one was not used anymore on the server side and we should remove it from the display